.github/workflows/build-and-push-docker-image.yml

```diff
@@ -3,6 +3,7 @@ on:
   push:
     branches:
       - 'dev-cere'
+      - 'feature/integrate-test-in-ci-flow'
 env:
```

Comment:
@huythanhcere 
You should use `on_pull_request` event. Please, read about it [here](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#configuring-workflow-events).

Reply:
Ok, i will change to use on_pull_request event in the next commit.

Reply:
Also, `on_pull_request` only tests should be executed.

Build should be on `push`